Sources/SafeDITool/SafeDITool.swift

```diff
@@ -30,7 +30,9 @@ import SwiftParser
 struct SafeDITool: AsyncParsableCommand {
     // MARK: Arguments
 
-    @Argument(help: "A path to a CSV file containing paths of Swift files to parse.") var swiftSourcesFilePath: String
+    @Argument(help: "A path to a CSV file containing paths of Swift files to parse.") var swiftSourcesFilePath: String?
+
+    @Option(parsing: .upToNextOption, help: "Directories containing Swift files to include, relative to the executing directory.") var include: [String] = []
```

Comment:
One of these days we may want to add an `exclude`, but for now this gets the job done.